guidance/diagrams/decision-tree-1.mmd

```diff
@@ -1,19 +1,19 @@
 flowchart TB
-	End(No extension)
+	End(No extension required)
```

Comment:
we need a stronger verb here, aligned with the context which is a binary decision.

Reply:
Addressed the comments for the Review and committed the changes. @zeitgeist